sycl/include/CL/sycl/range.hpp

```diff
@@ -141,6 +142,13 @@ template <int dimensions = 1> class range : public detail::array<dimensions> {
   __SYCL_GEN_OPT(^=)
 
 #undef __SYCL_GEN_OPT
+
+private:
+  friend class handler;
+  friend class detail::Builder;
+
+  // Adjust the first dim of the range
+  void set_range(const size_t dim0) { this->common_array[0] = dim0; }
```

Comment:
This sets first dim - better reflect this in the name (e.g. set_zero_dim)

Reply:
Changed name.